app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
```

Comment:
### [OPTIONAL]
Just like I explained in the post.rb, Please use the Rails Active Callbacks for comment.rb as well 👍🏻